src/Containers/BoxComposition/BoxComposition.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { BoxComposition, IBoxCompositionProps } from '@Containers';
+import { createStoryTitle } from '../../Constants';
+
+export default {
+    title: createStoryTitle('BoxComposition'),
+    component: BoxComposition,
+    args: {
+        data:[
+            {
```

Comment:
This structure won't work, 3 images are part of one set :) try to actually replace the content on the pulse then you will see why :)

Reply:
[{
mainImage: {imgSources: 'stringUrl',
                top:0,
                left:75,
                width:100,
                height:200,
                text:'bla bla bla',
                id:'0'},
leftImage: {},
rightImage: {}
}, {Set2}]